### PR TITLE
[18.0][MIG] connector_importer: Migration to 18.0

### PR DESCRIPTION
**Pull request by thienvh332**
### Depends on:
- https://github.com/OCA/queue/pull/692
- https://github.com/OCA/connector/pull/494

### CODE REVIEW

File: connector_importer/utils/mapper_utils.py [hunk 1]
```diff
@@ -284,11 +276,37 @@ def modifier(self, record, to_attr):
             # override by param
             search_operator = modifier.search_operator
 
-        # finally search it
         search_args = [(modifier.search_field, search_operator, search_value)]
-
         value = rel_model.search(search_args)
 
+        value = _handle_missing_values(
```
Comment (simahawk) [inline connector_importer/utils/mapper_utils.py:277]: can you please split this change to its own commit so that we can backport it?

**Review by simahawk — CHANGES_REQUESTED** [review]
There's  migration to v17 here... I guess the plan is to open a PR for that?

File: connector_importer/utils/mapper_utils.py (hunk 1 above)
Comment (simahawk) [reply connector_importer/utils/mapper_utils.py:277]: You can apply these changes on v16 if it eases your life and then we cherry-pick the commit  ;)

Comment (thienvh332) [reply connector_importer/utils/mapper_utils.py:277]: Thanks @simahawk ,
- I don't have plans to make a PR for v17. And I've isolated the modifier change out so you can cherry-pick it easily.
- Would you like me to create a PR to apply it to version 16?

Comment (simahawk) [reply connector_importer/utils/mapper_utils.py:277]: > Thanks @simahawk ,
> 
>     * I don't have plans to make a PR for v17. And I've isolated the modifier change out so you can cherry-pick it easily.

cool, tnx

>     * Would you like me to create a PR to apply it to version 16?

yes please, it will serve as a reminder :pray:

**Comment by simahawk** [comment]
a bunch of PRs to fwd port 

- https://github.com/OCA/connector-interfaces/pull/144
- https://github.com/OCA/connector-interfaces/pull/143
- https://github.com/OCA/connector-interfaces/pull/140

**Review by mmequignon — APPROVED** [review]

**Comment by simahawk** [comment]
TODO: rebase and drop test dep

**Comment by simahawk** [comment]
@thienvh332 can you drop test req? :)

**Comment by thienvh332** [comment]
> @thienvh332 can you drop test req? :)

Hi @simahawk , 
- We need **`odoo_test_helper`** to be able to pass unit test. So I added **`test-requirement`** with a normal commit. 
- Of course, If you will be ok when it says test failed. I will drop it.

**Comment by simahawk** [comment]
ahhh sorry, didn't check the content :sweat: 
Cool, thanks then!

**Review by simahawk — APPROVED** [review]
